level1c4pps/vgac2pps_lib.py

```diff
@@ -502,7 +572,14 @@ def process_one_scene(scene_files, out_path, engine="h5netcdf",
         sensor = "viirs"
         if noaa19_sbaf_version is not None:
             sensor = "avhrr"
-            convert_to_noaa19(scn_, noaa19_sbaf_version)
+            if "NN" not in noaa19_sbaf_version:
```

Comment:
I think the code will be more easy to test if you add this logic to the convert_to_noaa19 function. So I suggest that from this function you call  convert_to_noaa19(scn_, noaa19_sbaf_version) as before.

Reply:
OK. I think I understand and I moved the logic into convert_to_noaa19